Vagrantfile

```diff
@@ -0,0 +1,41 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "ubuntu/bionic64"
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update
+
+    #Install pyenv prerequisites
+    sudo apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+    #Install pyenv
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bashrc
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bashrc
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.bashrc
+    exec "$SHELL"
```

Comment:
Almost there but this line doesn't work. 

You can try creating the VM from scratch by running `vagrant destroy` followed by `vagrant up` and we would like it to create a VM that runs the app with just that up command.

`exec "$SHELL"` is for use in an interactive shell, to open a new one (with fresh environment variables etc). When run from the vagrant provisioning script (which is not run in an interactive shell), it actually just silently ends the script early so the pyenv commands below do not get run.

You should instead run `source ~/.profile`